### PR TITLE
From code camp, clarify NXmx/NXdetector depends_on field.

### LINKED ISSUE #747: NXmx/NXdetector depends_on field is undocumented
(discussion in #743)

### PR DESCRIPTION
**Pull request by phyy-nx**
Fixes #747

### CODE REVIEW

**Comment by phyy-nx** [comment]
I'd like this to be part of the next release.  @yayahjb can you review?

**Comment by prjemian** [comment]
Is there an issue related?  Needs to be edited into the description field above.

**Comment by prjemian** [comment]
I'm marking this for the milestone this week.  If it's not resolved by OCB Thursday, I'll move it to the next milestone.

File: applications/NXmx.nxdl.xml [hunk 1]
```diff
@@ -152,7 +152,14 @@
 						However, in the case of multiple detector elements, each element
 						needs a uniquely named NXdetector group.
 					</doc>
-					<field name="depends_on" type="NX_CHAR" /> <!-- better type for paths needed  -->
+					<field name="depends_on" type="NX_CHAR"> <!-- better type for paths needed  -->
+						<doc>
+							NeXus path to the detector positioner axis that most directly
+							supports the detector.  In the case of a multi-module detector,
+							this should be the most common axis, or ‘.’ if no axes are in
```
Comment (prjemian) [inline applications/NXmx.nxdl.xml]: The quote marks around the period look curly.  Likely to cause Sphinx problems.  Change to 

    ``.``

**Comment by phyy-nx** [comment]
Issue added and code review addressed.  I'll merge on Thursday if there are no objections.

**Pull request merged**